indexer/feature_decl.cpp

```diff
@@ -21,12 +21,14 @@ string DebugPrint(feature::EGeomType type)
   }
 }
 
-pair<FeatureID::MwmName, FeatureID::MwmVersion> FeatureID::GetMwmNameAndVersion() const
-{
-  if (!IsValid())
-    return {"INVALID", 0};
+constexpr char const * FeatureID::kInvalidFileName = "INVALID";
 
-  auto const & mwmInfo = m_mwmId.GetInfo();
-  return {mwmInfo->GetCountryName(), mwmInfo->GetVersion()};
+string FeatureID::GetMwmName() const
+{
+  return IsValid() ? m_mwmId.GetInfo()->GetCountryName() : kInvalidFileName;
 }
 
+int64_t FeatureID::GetMwmVersion() const
+{
+  return IsValid() ? m_mwmId.GetInfo()->GetVersion() : 0;
```

Comment:
Если тут int, то лучше -1 а не 0

> 24.3.2016, в 11.40, ygorshenin <notifications@github.com> написал(а):
> 
> In indexer/feature_decl.cpp:
> 
> >  }
> >  
> > +int64_t FeatureID::GetMwmVersion() const
> > +{
> > +  return IsValid() ? m_mwmId.GetInfo()->GetVersion() : 0;
> Здесь тоже лучше завести именованную константу типа kInvalidMwmVersion.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>